src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
 // write code here
+const rowCreate = document.querySelector('.append-row');
+const rowDelete = document.querySelector('.remove-row');
+const columnCreate = document.querySelector('.append-column');
+const columnDelete = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+
+// function for disable buttons, if rows or columns more or less then maxCount
+const MAX_COUNT = 10;
+const MIN_COUNT = 2;
+
+function updateButtonStates() {
+  const rowCount = document.querySelectorAll('.field tr').length;
+  const columnCount = document.querySelector('.field tr').children.length;
```

Comment:
The logic for determining the number of columns is incorrect. `document.querySelector('.field tr').children.length` assumes that there is always at least one row in the table. If the table is empty, this will cause an error. Consider adding a check to ensure there is at least one row before accessing `children.length`.